tools/pip/doc/CU112_ADDITIONAL.md

```diff
@@ -20,10 +20,10 @@ Prerequisites
 This package supports Linux and Windows platforms. You may also want to check:
 - [mxnet-cu110](https://pypi.python.org/pypi/mxnet-cu110/) with CUDA-11.0 support.
```

Comment:
Since this file is for cuda 11.2, we should have a line here for 11.2.

Reply:
We don't need it since this line actually tells about other variants that user may want to check out.

Reply:
Got it. Thanks.